py/desispec/pipeline/db.py

```diff
@@ -38,7 +38,7 @@ def all_task_types():
     """
     from . import tasks
     from .tasks.base import default_task_chain
-    ttypes = ["fibermap", "rawdata"]
+    ttypes = ["fibermap", "rawdata", "qadata"]
```

Comment:
I think you want to put "qadata" at the end of the list, so that in order we have:  fibermap, rawdata, <all other steps>, qadata.